app/views/curation_concerns/base/_attribute_rows.html.erb

```diff
@@ -1,11 +1,11 @@
-<%= presenter.attribute_to_html(:creator, catalog_search_link: true ) %>
-<%= presenter.attribute_to_html(:contributor, label: 'Contributors', catalog_search_link: true) %>
-<%= presenter.attribute_to_html(:subject, catalog_search_link: true) %>
-<%= presenter.attribute_to_html(:publisher) %>
-<%= presenter.attribute_to_html(:language) %>
-<%= presenter.attribute_to_html(:identifier) %>
-<%= presenter.attribute_to_html(:keyword) %>
-<%= presenter.attribute_to_html(:date_created) %>
-<%= presenter.attribute_to_html(:based_near) %>
-<%= presenter.attribute_to_html(:related_url) %>
-<%= presenter.attribute_to_html(:resource_type) %>
+<%= presenter.attribute_to_html(:creator, render_as: :faceted ) %>
```

Comment:
Did you try passing in the `search_field`? Not sure of exact syntax ATM, but perhaps something like:

```ruby
<%= presenter.attribute_to_html(:creator, render_as: :linked, search_field: 'creator_tesim') %>
```

Maybe the author of the `LinkedAttributeRenderer` can help.  @jcoyne ?